src/components/date-picker/dateFormatter.ts

```diff
@@ -0,0 +1,56 @@
+import 'moment/locale/da';
+import 'moment/locale/fi';
+import 'moment/locale/nb';
+import 'moment/locale/sv';
```

Comment:
This reminds me… we probably need to import French here too, right? And Dutch too soon.

(An issue for a different PR obviously. Just writing it down while I'm thinking of it.)